app/src/org/commcare/activities/FormEntryActivity.java

```diff
@@ -1533,6 +1545,41 @@ protected void onPause() {
         if (mLocationServiceIssueReceiver != null) {
             unregisterReceiver(mLocationServiceIssueReceiver);
         }
+
+        saveInlineVideoState();
+    }
+
+    private void saveInlineVideoState() {
+        for (int i = 0; i < questionsView.getWidgets().size(); i++) {
+            QuestionWidget q = questionsView.getWidgets().get(i);
+            if (q.findViewById(MediaLayout.INLINE_VIDEO_PANE_ID) != null) {
+                VideoView inlineVideo = (VideoView)q.findViewById(MediaLayout.INLINE_VIDEO_PANE_ID);
+                if (inlineVideo.isPlaying()) {
+                    indexOfWidgetWithVideoPlaying = i;
+                    positionOfVideoProgress = inlineVideo.getCurrentPosition();
+                    return;
+                }
+            }
+        }
+    }
+
+    private void restoreInlineVideoState() {
+        if (indexOfWidgetWithVideoPlaying != -1) {
+            QuestionWidget widgetWithVideoToResume = questionsView.getWidgets().get(indexOfWidgetWithVideoPlaying);
+            VideoView inlineVideo = (VideoView)widgetWithVideoToResume.findViewById(MediaLayout.INLINE_VIDEO_PANE_ID);
```

Comment:
Also confused, that's the point of having and using the `indexOfWidgetWithVideoPlaying` field